transactions/update-contract/2024/jun-13-update-service-account/FlowServiceAccount.cdc

```diff
@@ -0,0 +1,216 @@
+import FungibleToken from 0xf233dcee88fe0abe
```

Comment:
@janezpodhostnik - would love to get your 👁️ on this (since this is the service contract update)

Reply:
LGTM